src/method.jl

```diff
@@ -0,0 +1,116 @@
+"""
+    signature(meth::Method) -> Dict{Symbol,Any}
```

Comment:
Could we please not abbreviate `method` as `meth`?

Reply:
why not?
Problem with calling it `method` is that we can end up wanting to pluralize it.
but `methods` is a function which its better not to mask (unlike `meths`).
It probably would work here.

`meth` has been my habbitual abbreviation for this in numerous places.
Its what JuliaInterpetter does also:
https://github.com/JuliaDebug/JuliaInterpreter.jl/search?q=meth&type=

Reply:
`meths` is only used one place and could easily be `_methods`, which would allow us to use the more pleasant and more informative `method` all other places

Reply:
As these methods are fairly small I'm fine with `m` and `ms`

Reply:
ok